e2e/service/database/service.go

```diff
@@ -0,0 +1,125 @@
+package database
+
+import (
+	"encoding/hex"
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/repository"
+	"github.com/limechain/hedera-eth-bridge-validator/app/helper"
+	"github.com/limechain/hedera-eth-bridge-validator/app/helper/ethereum"
+	"github.com/limechain/hedera-eth-bridge-validator/app/model/auth-message"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/entity"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/message"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/transfer"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-eth-bridge-validator/e2e/model"
+	log "github.com/sirupsen/logrus"
+)
+
+type Service struct {
```

Comment:
I think that the db validation service should handle checks for all databases. This would mean that he should receive a config for the array of Db's and we simply do:
`database.VerifyTransferRecord`
`database.VerifyMessageRecords`